frontend/public/style/_overrides.scss

```diff
@@ -1,6 +1,12 @@
 // Use this file to override styles from 3rd party dependencies
 $pf-4-nav-bar-height: 76px; // Height of the PatternFly 4 masthead
 
+.alert-danger {
```

Comment:
I think it might be better to opt in with a class like `.co-alert--limit-height`. (I'm sure there's a better name.)

Reply:
I originally made new class `.alert--scrollable` but then thought we'd want to use it for all error alerts. Do we only want to apply it selectively?

Reply:
Switch to .co-alert-scrollable. ?